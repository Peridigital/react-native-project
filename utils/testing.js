import ShallowRenderer from 'react-test-renderer/shallow'

/**
 * Creates a shallow rendered string representation of a given react component
 *
 * @param {any} component - A React component to be snapshotted ex: <Text>Hello World<Text/>
 *
 * @returns {string} A shallow rendered string representation of the React component
 */
export function shallow (component) {
  const renderer = new ShallowRenderer()
  renderer.render(component)
  return renderer.getRenderOutput()
}
