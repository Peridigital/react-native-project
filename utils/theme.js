const colors = {
  primaryBackground: '#0C0D12',
  secondaryBackground: '#151821',
  primaryText: '#EAEAEA',
  border: '#7E4033',
  borderHighlight: '#D0A499'
}

export default colors
