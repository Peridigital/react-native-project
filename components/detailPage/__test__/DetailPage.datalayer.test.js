/* eslint-disable no-undef */
import React from 'react'
import { shallow } from '../../../utils/testing'
import DetailPageDatalayer from '../DetailPage.datalayer'

describe('DetailPageDatalayer', () => {
  it('should render', () => {
    const wrapper = shallow(<DetailPageDatalayer/>)

    expect(wrapper).toMatchSnapshot()
  })
})
