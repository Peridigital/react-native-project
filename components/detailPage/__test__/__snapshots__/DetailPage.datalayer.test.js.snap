// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailPageDatalayer should render 1`] = `<DetailPage />`;
